src/HIndent/Pretty.hs

```diff
@@ -1263,40 +1326,44 @@ instance Pretty Safety where
 
 --------------------------------------------------------------------------------
 -- * Unimplemented or incomplete printers
-
```

Comment:
We probably need this line. The comment should apply to a large file section, not to the instance